src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -144,12 +144,11 @@ Netex field | NTFS file | NTFS field | Note
 --- | --- | --- | ---
 Quay/@id | stops.txt | stop_id | see the section [Identifiers for Quay and StopPlace](#identifiers-for-quay-and-stopplace)
 Quay/@version | | | fixed value `any`.
-Quay/PublicCode | stops.txt | stop_code | This node may not be present if the stop_point has no `stop_code`.
-Quay/Name | stops.txt | stop_name | 
+Quay/Name | stops.txt | stop_name |
 Quay/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinates conversion](#coordinates-conversion)
-Quay/ParentSiteRef | stops.txt | parent_station | see the section [Identifiers for Quay and StopPlace](#identifiers-for-quay-and-stopplace)
 Quay/TransportMode | | | see (2) below
 Quay/tariffZones | stops.txt | fare_zone_id | The fare zone is prefixed by the `ParticipantRef` prefix with a `:` separator
```

Comment:
Quay/tariffZones/TariffZoneRef/@ref

Reply:
Thanks, done in 7b015b04663426e66cc7a63d0e9a5735364defb6.